python/tests/core/util/test_version.py

```diff
@@ -82,16 +88,25 @@ def test_ge(self):
         v2 = Version(1, 2, 0)
         self.assertFalse(v1 >= v2)
 
+        self.assertNotEqual(str(v1), str(v2))
+        self.assertNotEqual(repr(v1), repr(v2))
+
+
     def test_current(self):
         current = Version.currentVersion()
         self.assertTrue(current > (0, 0, 0))
+        pfx = 'Version(major='
+        self.assertEqual(pfx, repr(current)[:len(pfx)])
```

Comment:
Maybe test for all expected elements? Something along the lines of:

`fluff = "Version(major=, minor=, micro=, commit=, status=)"
self.assertEqual(fluff, re.sub(r'\d+|".+?"', "", s))`